meta-xt-domx-gen3/recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -1,19 +1,11 @@
 require xen-source.inc
 
-PACKAGES:append = "\
```

Comment:
Could you please clarify the commit message? It is unclear what "recent state" is

Reply:
I meant what we already have in meta-virtualization's kirkstone branch in the context of xen-tools.bb, will clarify

Reply:
"+"